ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -134,5 +129,332 @@ spec:
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Local Policies\Security Options\Accounts: Rename guest account'
+  query: |
+    SELECT 1 FROM mdm_bridge where mdm_command_input = "<SyncBody><Get><CmdID>1</CmdID><Item><Target><LocURI>./Device/Vendor/MSFT/Policy/Result/LocalPoliciesSecurityOptions/Accounts_RenameGuestAccount</LocURI></Target></Item></Get></SyncBody>" and mdm_command_output != "Guest";
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS-win10-stand-alone-1.1.7, CIS_Bullet_2.3.1.6
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Audit Force audit policy subcategory settings (Windows Vista or later) to override audit policy category settings' is set to 'Enabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting allows administrators to enable the more precise auditing capabilities.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via GP, set the following UI path to 'Enabled':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Local Policies\Security Options\Audit: Force audit policy subcategory settings (Windows Vista or later) to override audit policy category settings'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Lsa\\SCENoApplyLegacyAuditPolicy' AND data == 1);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS-win10-stand-alone-1.1.7, CIS_Bullet_2.3.2.1
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Audit Shut down system immediately if unable to log security audits' is set to 'Disabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting determines whether the system shuts down if it is unable to log Security
+    events. It is a requirement for Trusted Computer System Evaluation Criteria (TCSEC)-C2 and
+    Common Criteria certification to prevent auditable events from occurring if the audit system is
+    unable to log them. 
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via GP, set the following UI path to 'Disabled':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Local Policies\Security Options\Audit: Shut down system immediately if unable to log security audits'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Lsa\\crashonauditfail' AND data == 0);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS-win10-stand-alone-1.1.7, CIS_Bullet_2.3.2.2
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Devices Prevent users from installing printer drivers' is set to 'Enabled'
+  platforms: win10
+  platform: windows
+  description: |
```

Comment:
This applies to all items.
I am thinking why not put the whole description from the CIS doc?

e.g. For this item:


```
Description:
This policy setting determines who is allowed to format and eject removable NTFS media. You can use this policy setting to prevent unauthorized users from removing data on one computer to access it on another computer on which they have local administrator privileges.
The recommended state for this setting is: Administrators and Interactive Users. Rationale:
Users may be able to move data on removable disks to a different computer where they have administrative privileges. The user could then take ownership of any file, grant themselves full control, and view or modify any file. The fact that most removable storage devices will eject media by pressing a mechanical button diminishes the advantage of this policy setting.
Impact:
None - the default value is Administrators only. Administrators and Interactive Users will be able to format and eject removable NTFS media.

```

Reply:
Fixed as agreed